scipy/stats/tests/test_stats.py

```diff
@@ -3368,6 +3382,23 @@ def test_1samp_ci_iv(self):
         with pytest.raises(ValueError, match=message):
             res.confidence_interval(confidence_level=10)
 
+    shape = npst.array_shapes(min_dims=1, min_side=2)
+    alpha = strategies.floats(1e-8, 1-1e-8, allow_nan=False)
+    data = npst.arrays(dtype=npst.floating_dtypes(),
```

Comment:
```suggestion
    elements = dict(allow_nan=False, allow_infinity=False)
    data = npst.arrays(dtype=npst.floating_dtypes(), elements=elements,```

Reply:
I recommend spending your time thinking about the domain of possible inputs rather than the distribution, which in this case means leaving subnormals enabled.

The distribution is in any case a very complicated mixture of heuristics, feedback-guided search, replaying verbatim or modified examples from earlier, etc.